SIL.Windows.Forms.Keyboarding.Tests/XkbKeyboardAdapterTests.cs

```diff
@@ -400,6 +400,23 @@ public void ActivateKeyboard_SecondTime_NotCrash()
 			adaptor.SwitchingAdaptor.ActivateKeyboard(KeyboardController.Instance.Keyboards.First());
 			KeyboardController.Shutdown();
 		}
+
+		[Test]
+		public void CreateKeyboardDefinition()
+		{
+			// Setup
+			XklEngineResponder.SetGroupNames = new string[] { KeyboardUSA };
+			var adaptor = new XkbKeyboardRetrievingAdaptor(new XklEngineResponder());
+			KeyboardController.Initialize(adaptor);
+
+			// Exercise
+			var keyboard = XkbKeyboardRetrievingAdaptor.CreateKeyboardDefinition("/some/keyboard/without/dash",
```

Comment:
Is this supposed to look like a path? If not, `"KeyboardWithoutDash"` may be better.  Otherwise LGTM.